azure/notebooks/Azure-MNMG-XGBoost.ipynb

```diff
@@ -0,0 +1,1261 @@
+    "ManagedImageLocation: <some chosen location>\n",
+    "\n",
+    "```\n",
+    "    \n",
+    "</p>\n",
+    "</details>\n",
+    "\n",
+    "---\n",
+    "    \n",
+    "##### When `packer` finishes, at the bottom of the output, you will see something similar to the following:\n",
+    "```\n",
+    "ManagedImageResourceGroupName: <your resource group>\n",
+    "ManagedImageName: <your chosen custom image name>\n",
+    "ManagedImageId: /subscriptions/<your subscription id>/resourceGroups/<your resource group>/providers/Microsoft.Compute/images/<your chosen custom image name>\n",
+    "ManagedImageLocation: <some chosen location>\n",
+    "```\n",
+    "\n",
+    "##### Make note of the `ManagedImageId`. This is the resource id of the custom image we will use. \n",
+    "As shown above the `ManagedImageId` will look something like : `/subscriptions/12345/resourceGroups/myown-rg/providers/Microsoft.Compute/images/myCustomImage`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### f. Set up customized VM information and clear default dask config "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once you have the custom VM resource id, you should reset the default VM image information in `dask.config`. The default image value loaded in `dask.config` is that of a basic Ubuntu Server 18.04 LTS (the one that you already customized). If you do not reset it, `dask` will try to use that image instead of your custom made one."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ManagedImageId = <value from the output above> # or the customized VM id if you already have resource id of the customized VM from a previous run. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dask.config.set({\"cloudprovider.azure.azurevm.vm_image\":{}})\n",
+    "config = dask.config.get(\"cloudprovider.azure.azurevm\", {})\n",
+    "print(config)\n",
+    "vm_image = {\"id\": ManagedImageId}\n",
+    "print(vm_image)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### <span style=\"color:#8735fb; font-size:20pt\"> Step 1.2: Start the VM Cluster in Azure </span>\n",
+    "\n",
+    "Here, if you have used Option 1, i.e., the NVIDIA VM image, pass an empty string for `vm_image` information. \n",
+    "\n",
+    "For Option 2, pass the `vm_image` information that you got from the output of `packer` run as a parameter to `AzureVMCluster`. \n",
+    "\n",
+    "Also turn off the bootstrapping of the VM image by passing `bootstrap=False`. This will turn off installation of the dependencies in the VM, since we already have them on our custom VM image in either cases."
```

Comment:
```suggestion
    "Also turn off the bootstrapping of the VM by passing `bootstrap=False`. This will turn off installation of the dependencies in the VM, since we already have them on our custom VM in either cases."
```